assets/services.json

```diff
@@ -1298,6 +1298,25 @@
       ],
       "icon_svg": "<svg xmlns=\"http://www.w3.org/2000/svg\" fill=\"currentColor\" viewBox=\"0 0 30 30\"><path d=\"M 7 4 L 9 7.25 L 9 22.75 L 6.875 26 L 21.957031 26 L 25 22 L 14 22 L 14 4 L 7 4 z M 16 4.0507812 L 16 6.0585938 C 20.493 6.5575937 24 10.375 24 15 C 24 16.849 23.438516 18.569 22.478516 20 L 24.785156 20 C 25.556156 18.498 26 16.801 26 15 C 26 9.272 21.598 4.5577812 16 4.0507812 z M 6.8730469 7.6113281 C 5.0940469 9.5663281 4 12.155 4 15 C 4 17.837 5.0884219 20.418094 6.8574219 22.371094 L 7 22.154297 L 7 19.105469 C 6.365 17.872469 6 16.479 6 15 C 6 13.521 6.365 12.127531 7 10.894531 L 7 7.8164062 L 6.8730469 7.6113281 z\"/></svg>"
     },
+    {
+      "id": "line",
+      "name": "LINE",
+      "rules": [
+        "||gcld-line.com^",
+        "||lin.ee^",
+        "||line-apps-beta.com^",
+        "||line-apps-rc.com^",
+        "||line-apps.com^",
+        "||line-cdn.net^",
+        "||line-scdn.net^",
+        "||line.me^",
+        "||line.naver.jp^",
+        "||linecorp.com^",
+        "||linemyshop.com^",
+        "||lineshoppingseller.com^"
+      ],
+      "icon_svg": "<svg version=\"1.1\" fill=\"currentColor\" viewBox=\"0 0 320 320\" xmlns=\"http://www.w3.org/2000/svg\"><defs><style>.cls-1{fill:#06c755;}.cls-2{fill:#fff;}</style></defs><g data-name=\"Layer 2\"><g data-name=\"LINE LOGO\"><rect class=\"cls-1\" width=\"320\" height=\"320\" rx=\"72.14\"/><path class=\"cls-2\" d=\"m266.66 144.92c0-47.74-47.86-86.58-106.69-86.58s-106.69 38.84-106.69 86.58c0 42.8 38 78.65 89.22 85.42 3.48 0.75 8.21 2.29 9.4 5.26 1.08 2.7 0.71 6.93 0.35 9.65 0 0-1.25 7.53-1.52 9.13-0.47 2.7-2.15 10.55 9.24 5.76s61.44-36.18 83.82-61.95c15.46-16.95 22.87-34.19 22.87-53.27z\"/><path class=\"cls-1\" d=\"m231.16 172.49h-30a2 2 0 0 1-2-2v-46.55a2 2 0 0 1 2-2h30a2 2 0 0 1 2 2v7.57a2 2 0 0 1-2 2h-20.37v7.85h20.37a2 2 0 0 1 2 2v7.64a2 2 0 0 1-2 2h-20.37v7.86h20.37a2 2 0 0 1 2 2v7.56a2 2 0 0 1-2 2.07z\"/><path class=\"cls-1\" d=\"m120.29 172.49a2 2 0 0 0 2-2v-7.56a2 2 0 0 0-2-2h-20.37v-37a2 2 0 0 0-2-2h-7.6a2 2 0 0 0-2 2v46.53a2 2 0 0 0 2 2h30z\"/><rect class=\"cls-1\" x=\"128.73\" y=\"121.85\" width=\"11.64\" height=\"50.64\" rx=\"2.04\"/><path class=\"cls-1\" d=\"m189.84 121.85h-7.56a2 2 0 0 0-2 2v27.66l-21.3-28.77a1.2 1.2 0 0 0-0.17-0.21l-0.12-0.12-0.11-0.09h-0.06l-0.11-0.08h-0.06l-0.11-0.06h-8.47a2 2 0 0 0-2 2v46.56a2 2 0 0 0 2 2h7.57a2 2 0 0 0 2-2v-27.93l21.33 28.8a2 2 0 0 0 0.52 0.52l0.12 0.08h0.06l0.1 0.05h0.31a2.42 2.42 0 0 0 0.54 0.07h7.52a2 2 0 0 0 2-2v-46.44a2 2 0 0 0-2-2.04z\"/></g></g></svg>"
```

Comment:
Please use this instead:
```
<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 50 50" width="50px" height="50px">    <path d="M 9 4 C 6.24 4 4 6.24 4 9 L 4 41 C 4 43.76 6.24 46 9 46 L 41 46 C 43.76 46 46 43.76 46 41 L 46 9 C 46 6.24 43.76 4 41 4 L 9 4 z M 25 11 C 33.27 11 40 16.359219 40 22.949219 C 40 25.579219 38.959297 27.960781 36.779297 30.300781 C 35.209297 32.080781 32.660547 34.040156 30.310547 35.660156 C 27.960547 37.260156 25.8 38.519609 25 38.849609 C 24.68 38.979609 24.44 39.039062 24.25 39.039062 C 23.59 39.039062 23.649219 38.340781 23.699219 38.050781 C 23.739219 37.830781 23.919922 36.789063 23.919922 36.789062 C 23.969922 36.419063 24.019141 35.830937 23.869141 35.460938 C 23.699141 35.050938 23.029062 34.840234 22.539062 34.740234 C 15.339063 33.800234 10 28.849219 10 22.949219 C 10 16.359219 16.73 11 25 11 z M 23.992188 18.998047 C 23.488379 19.007393 23 19.391875 23 20 L 23 26 C 23 26.552 23.448 27 24 27 C 24.552 27 25 26.552 25 26 L 25 23.121094 L 27.185547 26.580078 C 27.751547 27.372078 29 26.973 29 26 L 29 20 C 29 19.448 28.552 19 28 19 C 27.448 19 27 19.448 27 20 L 27 23 L 24.814453 19.419922 C 24.602203 19.122922 24.294473 18.992439 23.992188 18.998047 z M 15 19 C 14.448 19 14 19.448 14 20 L 14 26 C 14 26.552 14.448 27 15 27 L 18 27 C 18.552 27 19 26.552 19 26 C 19 25.448 18.552 25 18 25 L 16 25 L 16 20 C 16 19.448 15.552 19 15 19 z M 21 19 C 20.448 19 20 19.448 20 20 L 20 26 C 20 26.552 20.448 27 21 27 C 21.552 27 22 26.552 22 26 L 22 20 C 22 19.448 21.552 19 21 19 z M 31 19 C 30.448 19 30 19.448 30 20 L 30 26 C 30 26.552 30.448 27 31 27 L 34 27 C 34.552 27 35 26.552 35 26 C 35 25.448 34.552 25 34 25 L 32 25 L 32 24 L 34 24 C 34.553 24 35 23.552 35 23 C 35 22.448 34.553 22 34 22 L 32 22 L 32 21 L 34 21 C 34.552 21 35 20.552 35 20 C 35 19.448 34.552 19 34 19 L 31 19 z"/></svg>
```

The one that you supplied has green color specified in it: `#06c755`

Reply:
My bad! :sweat_smile: I edited the logo with Inkscape and made the grayscale. I never noticed that Inkscape still left the original color in the SVG. 

I will update it.